### DISCUSSION TITLE
Spin up fan when stopped

### DISCUSSION THREAD

**Discussion by Bobika2002** [Rem0o/FanControl.Releases#1447] [Ideas]
Hello, would be possible to make feature like pwm control from bios has? When fan stop for some reason it will try to spin it up by pulses so it try if unsuccessful it will try again in 1 sec or whatever delay is set.

**Comment by Rem0o** [comment]
What's the use case? Why is the fan stopped? Why would it not start?

**Reply by Bobika2002** [reply]
I have 4 loud fans conected to radiator. The´re very loud so I want keep them at low rpm´s at idle but sometimes 1 of them stopps, fan control shows 0 but still treating it like spinning fan. Its not big deal just idea. (I have already adjusted settings so that it doesn't happen again) Thanks

**Reply by Rem0o** [reply]
Have you set your start and stop% properly on your controls?
